helloworld/helloworld_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helloworld
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	p "github.com/google/trillian-examples/helloworld/personality"
+)
+
+var (
+	trillianAddr = flag.String("trillian", "localhost:50054", "Host:port of Trillian Log RPC server")
+	treeID       = flag.Int64("tree_id", 0, "Tree ID")
+)
+
+// TestAppend appends a random entry to the log and ensures that the
+// checkpoint updates properly (locally on the personality's side).
+func TestAppend(t *testing.T) {
+	if *treeID == 0 {
+		t.Skip("--treeID flag unset, skipping test")
```

Comment:
```suggestion
		t.Skip("--tree_id flag unset, skipping test")
```

Same throughout the other tests.